doc/instances.md

```diff
@@ -57,6 +57,16 @@ How to import instances:
 :diataxis:Migrate from LXC </howto/migrate_from_lxc>
 ```
 
+```{only} diataxis
+How to pass an NVIDIA GPU to a container:
+```
+
+```{filtered-toctree}
+:titlesonly:
+
+:diataxis:Run docker GPU workloads </howto/container_gpu_passthrough_with_docker>
```

Comment:
The title is still too long unfortunately.
How about "Pass NVIDIA GPUs"? That's also closer to the topic title.

Reply:
You could add something about docker workloads to the introductory sentence.

Reply:
fixed